ultimatejobweb/templates/available_jobs.html

```diff
@@ -14,7 +14,7 @@
 </head>
 <body>
 <!--jumbotron-->
-<div class="container">
+<!--<div class="container">
```

Comment:
why are you commenting this out? If you don't want it, please delete it and keep the code clean.

Reply:
This is true for the other commented out sections as well